ui/components/app/signature-request-siwe/signature-request-siwe.js

```diff
@@ -101,38 +103,25 @@ export default function SignatureRequestSIWE({
       ) : null}
       <Message data={formatMessageParams(parsedMessage, t)} />
       {!isMatchingAddress && (
-        <ActionableMessage
-          className="signature-request-siwe__actionable-message"
-          type="warning"
-          message={t('SIWEAddressInvalid', [
+        <BannerAlert severity={SEVERITIES.WARNING}>
```

Comment:
Add margins 

<img width="575" alt="Screenshot 2023-03-18 at 10 17 03 AM" src="https://user-images.githubusercontent.com/8112138/226122702-04421ea7-6b74-4380-9df0-7337c1d0cb98.png">


```suggestion
        <BannerAlert severity={SEVERITIES.WARNING}
          marginLeft={4}
          marginRight={4}
          marginBottom={4}
        >
```

Reply:
oops, forgot to push up this change. Thank you!

Reply:
updated https://github.com/MetaMask/metamask-extension/pull/18207/commits/ee0ce10ebcf951bbc05b18197ca41dda4a534f37